### ISSUE TITLE
Add the appropriate file extension when downloading icons.

### ISSUE THREAD

**Issue by xcfrg** [MultiMC/MultiMC5#3702]
### Role

Edit/view icons that have been downloaded by the multimc launcher.

### Suggestion

Automatically add the file extension for the icon that being downloaded.

### Benefit

Small change that has no compromises that I can see, but makes it easier to view or edit icons that have been downloaded, without having to edit the file extension first.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by NewoIsTaken** [comment]
I was looking through the issues and found this one. I have not worked on MultiMC before but this issue seems like a good starter issue. Could someone point to me the things that would need to be changed for this to happen?

**Comment by phit** [comment]
> I have not worked on MultiMC before but this issue seems like a good starter issue. Could someone point to me the things that would need to be changed for this to happen?

indeed, look for setSuggestedIconFromFile in the source and go from there!

**Comment by NewoIsTaken** [comment]
@phit Hey! Would you like to connect in Discord?

**Comment by phit** [comment]
just join the MultiMC discord, and we are happy to provide guidance in there if you are interested in contributing https://discord.gg/multimc
